builtin/providers/openstack/resource_openstack_networking_secgroup_rule_v2_test.go

```diff
@@ -85,6 +85,115 @@ func TestAccNetworkingV2SecGroupRule_timeout(t *testing.T) {
 	})
 }
 
+func TestAccNetworkingV2SecGroupRule_vrrp(t *testing.T) {
```

Comment:
I think we should change the test name to something like `TestAccNetworkingV2SecGroupRule_protocols` since it's testing more than just VRRP.